### PR TITLE
TOC reorganization and additional descriptions for model zoo README

### PR DESCRIPTION
**Pull request by vinitra**
- grouping logical categories together (in anticipation of more models being added to the model zoo)
i.e. semantic segmentation, object detection and segmentation -> object detection and image segmentation
- adding more detailed descriptions for image classification models
- adding descriptions of new model categories

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Open Neural Network eXchange (ONNX) Model Zoo
-
-[![Generic badge](https://img.shields.io/badge/Status-Work_In_Progress-red.svg)](#) 
 [![Generic badge](https://img.shields.io/badge/Contributions-Welcome-brightgreen.svg)](contribute.md)
+[![Generic badge](https://img.shields.io/badge/Status-Work_In_Progress-red.svg)](#)
```
Comment (prasanthpul) [inline README.md]: Everything is always "in progress". Suggest removing this.

File: README.md [hunk 2]
```diff
@@ -16,16 +15,11 @@ The Open Neural Network eXchange ([ONNX](http://onnx.ai)) is an open format to r
...
-* [Semantic Segmentation](#semantic_segmentation)
-* [Object Detection & Segmentation](#object_detection)
-* [Face Detection and Recognition](#face_detection)
-* [Emotion Recognition](#emotion_detection)
-* [Gender Detection](#gender_detection)
-* [Hand Written Digit Recognition](#digit_recognition)
-* [Super Resolution](#super_resolution)
+* [Object Detection & Image Segmentation](#object_detection)
+* [Body, Face & Gesture Analysis](#body_analysis)
 * [Style Transfer](#style_transfer)
```
Comment (prasanthpul) [inline README.md]: Are there a lot of style transfer models? If not maybe have a "image manipulation" category that includes Style transfer and Super resolution.

File: README.md [hunk 3]
```diff
@@ -73,51 +65,35 @@ These models detect the presence of multiple objects in an image and segment out
...
 |<b>YOLO v3</b>|[Redmon et al.](https://pjreddie.com/media/files/papers/YOLOv3.pdf)|[contribute](contribute.md)|
+|<b>[DUC](models/semantic_segmentation/DUC/)</b>|[Wang et al.](https://arxiv.org/abs/1702.08502)|Deep CNN based semantic segmentation model with >80% [mIOU](/models/semantic_segmentation/DUC/README.md/#metric) (mean Intersection Over Union), trained on urban street images|
+|<b>FCN</b>|[Long et al.](https://people.eecs.berkeley.edu/~jonlong/long_shelhamer_fcn.pdf)|[contribute](contribute.md)|
 
 <hr>
 
-### Face Detection and Recognition <a name="face_detection"/>
-These models detect and/or recognize human faces in images. Some more popular models are used for detection/recognition of celebrity faces, gender, age, and emotions.
+### Body, Face & Gesture Analysis <a name="body_analysis"/>
+Face detection models indentify and/or recognize human faces in images. Some more popular models are used for detection of celebrity faces, gender, age, and emotions.
```
Comment (prasanthpul) [inline README.md]: indentify -> identify

File: README.md [hunk 4]
```diff
@@ -73,51 +65,35 @@ These models detect the presence of multiple objects in an image and segment out
...
-
-|Model Class |Reference |Description |
-|-|-|-|
+|[**Emotion FerPlus**](emotion_ferplus) |[Barsoum et al.](https://arxiv.org/abs/1608.01041)	| Deep CNN for emotion recognition trained on images of faces.|
 |Age and Gender Classification using Convolutional Neural Networks| [Levi et al.](https://www.openu.ac.il/home/hassner/projects/cnn_agegender/CNN_AgeGenderEstimation.pdf)	|[contribute](contribute.md)|
 <hr>
 
-### Hand Written Digit Recognition <a name="digit_recognition"/>
-
+### Style Transfer <a name="style_transfer"/>
```
Comment (prasanthpul) [inline README.md]: needs description like the other categories

File: README.md (hunk 1 above)
Comment (vinitra) [reply README.md]: Removed!

File: README.md (hunk 3 above)
Comment (vinitra) [reply README.md]: Thanks for catching the typo!

File: README.md (hunk 4 above)
Comment (vinitra) [reply README.md]: Added for the remaining categories in new update.

**Pull request merged**